coredns/datadog_checks/coredns/data/auto_conf.yaml

```diff
@@ -0,0 +1,35 @@
+ad_identifiers:
+  - coredns/coredns
```

Comment:
The `ad_identifier` identifier for auto_conf needs to be the short name for container images. I think this needs to be

```
ad_identifiers:
  - coredns
```